src/lib/games/constants.ts

```diff
@@ -20,12 +20,20 @@ export const GAME_MODES = [
   "fewest-characters-to-llm",
 ] as const
 
-export type QuestionDifficultyLevels = (typeof questionDifficultyEnum)['enumValues'][number]
+export type QuestionDifficultyLevels = (typeof questionDifficultyEnum)["enumValues"][number]
 // this isnt a strict satisfies need to think how to do it strictly
-export const QUESTION_DIFFICULTY_LEVELS = ['easy', 'medium', 'hard'] as const satisfies QuestionDifficultyLevels[];
+export const QUESTION_DIFFICULTY_LEVELS = [
+  "easy",
+  "medium",
+  "hard",
+] as const satisfies QuestionDifficultyLevels[]
 
-export type GameStatus = (typeof gameStatusEnum)['enumValues'][number]
-export const GAME_STATUS = ['waitingForPlayers', 'inProgress', 'finished'] as const satisfies GameStatus[];
+export type GameStatus = (typeof gameStatusEnum)["enumValues"][number]
+export const GAME_STATUS = [
+  "waitingForPlayers",
+  "inProgress",
+  "finished",
+] as const satisfies GameStatus[]
```

Comment:
I am using prettier here, I think this formatting was when I didn't have prettier configured properly